### PR TITLE
Bundle react-components using Webpack

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Chromatic

https://837-react-webpack--60f9b557105290003b387cd5.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/837

## Acceptance criteria
- [x] `react-components` package is bundled using Webpack
    - [x] Bundle should be in `/dist`
- [x] `component-library` is updated to use the the `react-components` bundle file
- [ ] `component-library` is tested and working in `vets-website`

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by pdavies88 — APPROVED** [review]

**Comment by pdavies88** [comment]
@andresriveratoro I'm assuming we will end up closing #413 and #412 once this is merged in

File: packages/storybook/stories/Modal.stories.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import Modal from '../../react-components/src/components/Modal/Modal';
+import { Modal } from '@department-of-veterans-affairs/component-library';
```
Comment (k80bowman) [inline packages/storybook/stories/Modal.stories.jsx:2]: Would all of the imports in vets-website need to be updated to look like this as well?

Comment (andresriveratoro) [reply packages/storybook/stories/Modal.stories.jsx:2]: This is optional now that I've reverted the changes where the React components were no longer copied.

**Comment by andresriveratoro** [comment]
`react-components` was tested locally on a new React project and was working as expected except for the `Banner` component that utilizes the `VaAlert` react binding.

`component-library` was tested locally on a new React project. React components are working with the exception of `Banner`, but web components and React bindings do not work. The issue that I'm encountering happens when calling the `defineCustomElements` function.

Screen Shot 2022-05-31 at 11 39 20

Possibly related issues: https://github.com/ionic-team/stencil/issues/1882

**Comment by bkjohnson** [comment]
Closing due to this comment: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/837#issuecomment-1258565203

**Pull request closed without merging**